client/src/index.tsx

```diff
@@ -31,6 +33,8 @@ ReactDOM.render(
                     <Route component={DashboardContainer}>
                         <IndexRoute component={HomeContainer} />
                         <Route path='lesson/:id' component={LessonCreatorContainer} />
+                        <Route path='course/:id' component={CourseContainer} />
```

Comment:
Can we do lesson/:id and lesson/:id/edit for study and creator respectively?

You'll also need to edit the push call in createLesson in lesson.ts